.github/workflows/cd.yml

```diff
@@ -1,11 +1,48 @@
-name: Build Apps and Create Release
+name: Build Apps, Deploy Web and Create Release
 
 on:
   push:
     tags:
       - "[0-9]+.[0-9]+.[0-9]+"
 
 jobs:
+  build_web:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: corepack enable
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: pnpm
+
+      - name: 📦 Install dependencies
+        run: pnpm install  --frozen-lockfile
+
+      - name: Build dist bundle
+        run: pnpm build
+
+      - name: Upload dist bundle to S3
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          SOURCE_DIR: dist
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: Compress artifacts
+        run: |
+          zip -r Voyager-Web-${{ github.ref_name }}-Build.zip dist
```

Comment:
`server.mjs` not included in zip file?

Reply:
No, this is just the static bundle. `server.mjs` is not needed and actually will be removed soon :)

vger.app now runs in a plain s3 bucket.